wrappers/python/simtk/openmm/app/forcefield.py

```diff
@@ -888,7 +888,23 @@ def _getResidueTemplateMatches(self, res, bondedToAtom, templateSignatures=None,
                 template = allMatches[0][0]
```

Comment:
Line 869 lists the incorrect type in the docstring:
```
        template : _ForceFieldTemplate
            The matching forcefield residue template, or None if no matches are found.
```
`_ForceFieldTemplate` should be `_TemplateData`.